app/services/editions/publishing/events/PublishEventsQueueFacade.scala

```diff
@@ -45,22 +45,22 @@ private[events] class PublishEventsSQSFacade(val config: ApplicationConfiguratio
     }
   }
 
-  private def receiveMessages: List[Message] = {
+  private def receiveMessage: Option[Message] = {
     Try {
       val receiveRequest = new ReceiveMessageRequest()
         .withQueueUrl(queueURL)
         .withMaxNumberOfMessages(maxNumberOfSQSMessagesPerReceiveReq)
         .withWaitTimeSeconds(sqsClientLongPoolingWaitTimeSec)
```

Comment:
Just for my own curiosity -- `maxNumberOfSQSMessagesPerReceiveReq` looks like it doesn't have to be 1, but we seem to only be parsing the head of this list. Is there a possibility we could drop messages here?

Reply:
Great spot and I should have made the PR description more explicit to justify my reasoning for this change.

We're reading from the queue, parsing into case class then writing to database. Writing multiple records to postgres is totally fine, however the query is slightly more complex than writing a single record.

Given we're only ever reading one message from SQS, moving to an `Option[Message]` rather than a `List[Message]` massively simplifies the code and also forces us to carefully consider the impact of increasing the value of `maxNumberOfSQSMessagesPerReceiveReq` in the future.

Reply:
yeah, unfortunately long pooling is more confusing :) 
We are getting one message per req continuously
and when SQS become empty for given client it will wait 15 seconds until it will do next call
It's kind of like scheduling with 15 sec interval but i bit more reactive
and making the interface explicitly to get 1 message make sense